raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Duty to keep abreast with technological changes 
+(111) [1] It shall be the duty of the Council in collaboration with any other organ to closely follow developments in technology and techniques for preventing or reducing pollution of the environment due to releases of substances from prescribed processes. [2] Any person carrying out activity impacting on the environment shall be required to follow a directive that may be issued by the Council on adaptation of any such developments in technology and techniques to prevent or reduce pollution of the environment. 
+
+Duty to disclose information to prevent pollution 
+(112) [1] For purposes of facilitating pollution prevention and control measures, the Minister may prescribe rules obliging submission of information to either the Council or any designated person, by: [a] a person who owns or operates an irrigation project scheme, sewerage system, industrial production plant, industrial wastes treatment plant, solid waste disposal site or facility, trading and storage facility for petroleum products, workshop or any other undertaking which discharges or is likely to discharge effluents or other pollutants into the environment; or [b] any other institution or organ which is required by any other written law to take measures to prevent pollution within areas to which it has administrative control or any other institution or organ that may be identified from time to time. [2] The provisions of subsection [1] shall not derogate powers of the Minister responsible for shipping to make regulation for prevention of marine pollution pursuant to the provisions of Part XIX of the Merchant Shipping Act, 2003. [3] Any person who contravenes the requirement to furnish information to the Council or any designated person, commits an offence. 
+
+Prohibition notices on prescribed process or activity 
+(113) [1] Where the Council is of the opinion that the carrying on of a prescribed activity or process or the continuing carrying on of any process or activity in a particular manner involves an imminent risk of serious pollution of the environment, the Council may serve a prohibition notice on the person carrying on the process or activity. [2] A prohibition notice issued under subsection [1] shall: [a] specify the environmental risk involved in the activity or process; [b] specify the steps that may be taken to the process or activity to remove it and the period within which such steps shall be taken; [c] direct that the authorization shall, until the notice is withdrawn partly or wholly to the extent specified in the notice ceases, to have effect to authorize the carrying on of the process or activity; and [d] where, it applies only to a part of the process or activity it imposes conditions to be observed in carrying on that other process or activity which is authorized to be carried on. 
```

Comment:
Please replace:
* `ceases, to have effect` with `ceases to have effect`; and
* `where, it applies` with `where it applies`.